### ISSUE TITLE
Add qr code phising

### ISSUE THREAD

**Issue by Aspect0002** [fairnightzz/Discord-Login-Phishing#6]
Hey, can you add qr code generator and if people scanning it, it will grab their token and it send it to discord webhook

**Comment by fairnightzz** [comment]
I'm not too sure how to go about doing that, if you could elaborate a little more on how it works

**Comment by FuckingToasters** [comment]
> I'm not too sure how to go about doing that, if you could elaborate a little more on how it works

you can find python scripts for that on github and the javascript libary "brython" let you add python code in a html body

**Comment by fairnightzz** [comment]
Python doesn't need to necessarily be "used" since any language would be able perform the function to a certain extent. What I meant was that I am unfamiliar with **generating the qr code** as well as having Discord verify that qr code. The webhook is no problem, the main problem is generating a qr code that Discord can validate.

**Comment by FuckingToasters** [comment]
> 

Python is a easy language where even noobs can understand the sourcecode. Generating the Code is not hard i guess when using selenium BUT selenium is not the best option. Requests would be better because with requests, you could generate it faster and maybe add it in a for _ in ... loop or make it so after the code expired generating a new one etc.

**Comment by fairnightzz** [comment]
I don't think you understand what I mean. Using Python or JavaScript isn't the issue, because both languages can perform the task you're describing (for Python you mention selenium and requests, the same function can be performed with selenium/puppeteer and axios/requests). However, I'm not too sure as to how to generate the qr code - it seems to me that you intend to scrape it from the official login site? I can't just make a random we code because discord won't be able to validate it - the qr code is essentially a link they created themselves. If you have an understanding of how to get a valid qr code then by all means share it. Additionally, JavaScript is by no means a hard language (very similar to Python) and if you have an idea, feel free to contribute and file a pull request.

**Comment by fairnightzz** [comment]
Closed due to not enough info
